samples/apps/copilot-chat-app/WebApp/src/App.tsx

```diff
@@ -60,9 +60,13 @@ const App: FC = () => {
     useEffect(() => {
         if (isAuthenticated && account && appState === AppState.LoadingChats) {            
             // Load all chats from memory
-            chat.loadChats().then(() => {
-                setAppState(AppState.Chat);
-            });
+            async function loadChats() {
+                if (await chat.loadChats()) {
```

Comment:
Ideally, we should addAlert() to set the alert state of the app so the UI will render error banners to show users failure on critical path. But I'm okay if we want to address it next time considering the bug is painful to users.

Reply:
The alert state is populated inside the loadChats call.